### PR TITLE
PDS-3802 PDS-3933 PDS-3793 PDS-3791

### PR DESCRIPTION
**Pull request by ROMzombie**
Catch up with main and apply a few versions to fix PDS vulnerabilities.

### CODE REVIEW

File: v1/pom.xml [hunk 1]
```diff
@@ -218,7 +215,7 @@
       <version>${hadoop.version}</version>
       <exclusions>
         <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
+          <groupId>com.fasterxml.jackson</groupId>
```
Comment (ROMzombie) [inline v1/pom.xml:218]: This dependency moved and all the versions on the old naming have the vulnerability, so moving to the new repository name.

File: v1/pom.xml [hunk 2]
```diff
@@ -45,7 +44,7 @@
     <protobuf.version>3.21.12</protobuf.version>
     <nashorn.version>15.4</nashorn.version>
     <junit.jupiter.version>5.5.2</junit.jupiter.version>
-    <codehaus-jackson.version>1.9.13</codehaus-jackson.version>
+    <codehaus-jackson.version>2.14.0</codehaus-jackson.version>
```
Comment (ROMzombie) [inline v1/pom.xml:47]: Needing to pin this to the current com.fasterxml version.

File: v1/pom.xml [hunk 3]
```diff
@@ -431,16 +432,16 @@
...
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value-annotations</artifactId>
-      <version>${autovalue.annotations.version}</version>
+      <version>${autovalue.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.codehaus.jackson</groupId>
-      <artifactId>jackson-core-asl</artifactId>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
```
Comment (ROMzombie) [inline v1/pom.xml:439]: This dependency moved and all the versions on the old naming have the vulnerability, so moving to the new repository name.

File: v1/pom.xml [hunk 4]
```diff
@@ -431,16 +432,16 @@
...
       <version>${codehaus-jackson.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.codehaus.jackson</groupId>
-      <artifactId>jackson-mapper-asl</artifactId>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```
Comment (ROMzombie) [inline v1/pom.xml:444]: This dependency moved and all the versions on the old naming have the vulnerability, so moving to the new repository name.

**Review by bbooth — APPROVED** [review]

**Pull request merged**